lib/pluginmanager/new.rb

```diff
@@ -0,0 +1,103 @@
+# encoding: utf-8
+require "pluginmanager/command"
+require "pluginmanager/templates/render_context"
+require "erb"
+require "ostruct"
+require "fileutils"
+
+class LogStash::PluginManager::New < LogStash::PluginManager::Command
+
+  parameter "TYPE",   "Type of the plugin {input, codec, filter, output}s", :attribute_name => :type
+  parameter "PLUGIN", "Name of the new plugin", :attribute_name => :plugin_name
+
+  TYPES = [ "input", "filter", "output" ]
+
+  def execute
+    validate_parameters
+
+    source = File.join(File.dirname(__FILE__), "templates", "#{type}-plugin")
+    target = FileUtils.mkdir(File.join(Dir.pwd, full_plugin_name))
+    puts " #{ green("Creating") } #{full_plugin_name}"
+
+    begin
+      create_scaffold(source, target)
```

Comment:
@ph why? not sure we have to provide this kind of information here.also adds a new layer of maintenance, if the URL changes in the website then. Even if I like the idea, not sure to provide this kind of feedback here.